src/sentry/tasks/post_process.py

```diff
@@ -564,6 +446,160 @@ def process_snoozes(group):
     return False
 
 
+def process_event(data, group_id):
```

Comment:
Would it make sense to add typing to the newly created methods?

Reply:
sure thing